Library/Homebrew/standalone/sorbet.rb

```diff
@@ -2,6 +2,7 @@
 # frozen_string_literal: true
 
 require "sorbet-runtime"
+require "extend/module"
```

Comment:
This is mostly unrelated, but as of [this](https://github.com/Homebrew/brew/pull/16406/files#diff-eb8b4c210b5cc8f5d6cbbedc40fac8201d598efc5d4310002a9066d0b6ce4902) change, the extension only extends `Module` to `include T::Sig`, so it makes the most sense to live here IMO.